docs/proposal/20230113-capibm-cli-tool.md

```diff
@@ -0,0 +1,132 @@
+# CLI tool for Cluster API Provider IBM Cloud
+
+## Summary
+
+This proposal aims to provide a cli tool to help users to perform various IBM Cloud operations which helps in creating and
+managing Power VS or VPC workload clusters.
+
+## Motivation
+
+In order to create IBM Cloud Power VS cluster using cluster api as a prerequisite we need to create Power VS network, port and need import the appropriate image to Power VS service instance, currently we need to make use of different tools(ibmcloud, pvsadm cli) to achieve this.
+
+The goal is to create a single unified cli tool to handle all the operations required prior creating a kubernetes cluster using cluster-api-provider-ibmcloud.
+
+## Proposed commands
+
+Command will be of following format
+
+```shell
+capibmadm <provider> <command> <subcommand>
+```
+Here provider can be either vpc or powervs
+
+## Power VS Commands
+
+**Environment Variables:**
+
+  * IBMCLOUD_API_KEY: IBM Cloud api key of user.
```

Comment:
can we move `--service-instance-id` flag here as a common flag to all the subcommands to powervs?

Reply:
Move it as a common flag.